pysoa/test/stub_service.py

```diff
@@ -184,6 +182,19 @@ def stub_action(self, action, body=None, errors=None):
         self.action_class_map[action] = _make_stub_action(action, body, errors)
 
 
+class _StubActionRequestCounter(object):
+    def __init__(self):
+        self._counter = 0
+
+    def get_next(self):
+        value = self._counter
+        self._counter += 1
```

Comment:
Do we need to consider thread safety here?

Reply:
No, because this is used exclusively for unit tests, which are run single-threaded.